src/Native/Runtime/arm64/PInvoke.S

```diff
@@ -153,7 +153,12 @@ NoAbort:
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////
     LEAF_ENTRY RhpReversePInvoke, _TEXT
 
-        INLINE_GETTHREAD x10    // x10 = Thread
+#ifdef FEATURE_EMULATED_TLS
+        GETTHREAD_ETLS_10
```

Comment:
`RhpReversePInvoke` is not used by CoreRT. It is a left-over from .NET Native for UWP. It may be best to put the whole function under `#ifndef FEATURE_EMULATED_TLS` or even delete it.

Reply:
I removed it. If it ever needed again it's still in the history.